examples/send-onetimekey-tx/app.ts

```diff
@@ -0,0 +1,35 @@
+// tslint:disable: no-console
+
+import { Producer } from '../../src'
+
+export const start = async () => {
+  try {
+    const producer = new Producer(['localhost:9092'])
+    await producer.connect()
+
+    // Deploy a new SimpleToken contract
+    // const envelopeId = await producer.sendTransaction({
+    //   chainName: 'besu',
+    //   contractName: 'SimpleToken',
+    //   methodSignature: 'transfer(address,uint256)',
+    //   args: ['0x7e654d251da770a068413677967f6d3ea2fea9e5', 5000],
+    //   to: '0xe5ce65038f9d1c841a33CC816eE674F8a0E31E74',
+    //   contextLabels: {
+    //     txFrom: 'one-time-key'
+    //   }
+    // })
```

Comment:
You could remove this comment as one time key wont work with this smart contract

Reply:
Okay, updated